src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Authoring/Microsoft.Azure.CognitiveServices.LUIS.Authoring.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring</PackageId>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Authoring API.</Description>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;LUIS;Language Understanding Intelligent Service;Language Understanding;REST HTTP client</PackageTags>
     <PackageReleaseNotes>
```

Comment:
`PackageReleaseNotes` must be updated

Reply:
@dsgouda I edited the package release notes. But I do not know what else to add, because the only change in this PR is customizing the endpoint